src/components/general/reportOverview.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { View, Text, TouchableHighlight, StyleSheet } from 'react-native'
+
+export default function ReportOverview(props) {
+    return (
+        <View style={style.reportDetails}>
+            <View>
+                <Text>🤬</Text>
```

Comment:
I recommend using constants instead of direct icons.

Example:

```
const ICON_ANGRY = 🤬;
const ICON_LIKE = 👍;
const ICON_SEE = 👁️;

 <Text>{ ICON_ANGRY }</Text>
```

Reply:
I tried that, but you can't asign an emoji as a value.

![image](https://user-images.githubusercontent.com/33968113/120229062-fc6a7580-c211-11eb-8158-5c7ccc069a5f.png)